### ISSUE TITLE
Error loading regenerator.js

### ISSUE THREAD

**Issue by thesbros** [systemjs/builder#122]
When using code that has generators w/ babel, I get the following error:
```js
Error loading "npm:babel-runtime@4.7.16/regenerator" at file:**PROJECT**/app/public/jspm_packages/npm/babel-runtime@4.7.16/regenerator.js
Error loading "npm:babel-runtime@4.7.16/regenerator" from "app/index" at file:**PROJECT**/app/scripts/index.js
ENOENT: no such file or directory, open '**PROJECT**/app/public/jspm_packages/npm/babel-runtime@4.7.16/regenerator.js']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '**PROJECT**/app/public/jspm_packages/npm/babel-runtime@4.7.16/regenerator.js'
```
If I create the file "app/public/jspm_packages/npm/babel-runtime@4.7.16/regenerator.js" and add the following to it:
```js
module.exports = require("./regenerator");
```
It works fine.

Using systemjs-builder 0.10.2.

**Comment by guybedford** [comment]
Thanks so much for spotting this! I've added a fix in https://github.com/jspm/registry/commit/8060d3cba3689973eca25b0d97e030b88fd80eca and https://github.com/systemjs/builder/commit/34a86abff8b0190931717bf1fd3a9b6ee135d3e4, which is released in 0.10.3, jspm release to follow.

**Comment by thesbros** [comment]
No problem, thanks!
